### PR TITLE
Use implicit animations for animatable properties.

### PR DESCRIPTION
**Pull request by squall77it**
Tweener was replaced with Clutter implicit animations.

### CODE REVIEW

**Comment by squall77it** [comment]
Sorry I want to make a second pull request with Italian translation, but the commit was automatically added to this pull request, i revert it. When this is closed i make a new one. I'm not very good with git.

File: ShutdownTimer@neumann/extension.js [hunk 1]
```diff
@@ -49,11 +50,11 @@ function _showTextbox(textmsg) {
...
-	Tweener.addTween(textbox,
-		             { opacity: 0,
-		               time: 4,
-		               transition: 'easeOutQuad',
-		               onComplete: _hideTextbox });
+	textbox.ease ({
+		opacity: 0,
+		duration: 4000,
+		mode: Clutter.AnimationMode.EASE_OUT_QUAD,
+		onComplete:  () => { _hideTextbox; }
```
Comment (neumann-d) [inline ShutdownTimer@neumann/extension.js:57]:
```suggestion
	 textbox.ease ({
		opacity: 0,
		duration: 4000,
		mode: Clutter.AnimationMode.EASE_OUT_QUAD,
		onComplete:  _hideTextbox
	});
```

**Comment by neumann-d** [comment]
Thank you very much @squall77it  for this pull request. I'm still running GNOME 3.36, but your code change also works here. I just got some Syntax Error because of missing closing brackets. See my suggestion. When you update your Pull Request, I will merge it :+1:

**Comment by squall77it** [comment]
Thank you for the great extension. Sorry for the  syntax error, but when i cleaned  the file I deleted one line too much.
I have updated the pull request.

**Comment by neumann-d** [comment]
Thank you  @squall77it for contributing to this extension

**Pull request merged**